include/input.hpp

```diff
@@ -223,8 +223,9 @@ class VSInput {
   struct FreeEnergyIntegrand {
     std::vector<double> grid;
     std::vector<std::vector<double>> integrand;
+    std::vector<double> alphaData;

```

Comment:
I would call this just alpha or freeParameter. Once you change the name make sure that you are consistent in the wrappers, in python and in the tests